docs/pre.sh

```diff
@@ -3,40 +3,40 @@
 ## Create modules pages
 for D in ../x/*; do
   if [ -d "${D}" ]; then
-    MODDOC=docs/modules/$(echo $D | awk -F/ '{print $NF}')
+    MODDOC=docs/integrate/modules/$(basename "$D")
     rm -rf $MODDOC
-    mkdir -p $MODDOC && cp -r $D/README.md "$_"
+    mkdir -p $MODDOC && cp -r $D/README.md "$MODDOC/$(basename "$D").md"
   fi
 done
 
 ## Vesting is a submodule of auth, but we still want to display it in docs
 ## TODO to be removed in https://github.com/cosmos/cosmos-sdk/issues/9958
-cp ../x/auth/vesting/README.md ./docs/modules/auth/1-vesting.md
-cp ../x/auth/tx/README.md ./docs/modules/auth/2-tx.md
+cp ../x/auth/vesting/README.md ./docs/integrate/modules/auth/1-vesting.md
+cp ../x/auth/tx/README.md ./docs/integrate/modules/auth/2-tx.md
 
 ## Add modules page list
-cat ../x/README.md | sed 's/\.\.\/docs\/building-modules\/README\.md/\/building-modules\/intro\.html/g' > ./docs/modules/README.md
+cat ../x/README.md | sed 's/\.\.\/docs\/integrate\/building-modules\/README\.md/\/building-modules\/intro\.html/g' > ./docs/integrate/modules/"$MODDOC/$(basename "$D").md"
```

Comment:
Shouldn't it just be `./docs/integrate/modules/README.md`. What means that `$MODDOC` variable here?

Reply:
its so that when the docs are transferred over to the docs-repo the name of the page doesnt read "README" and reads the name of the previous file. So the `$MODDOC` is representative of the path e.g. `docs/integrate/modules/auth/auth.md`

Reply:
Yeah but here what is the value of $MODDOC and $D here.
Maybe I am reading this wrong but I think it takes the last value of whatever was $MODDOC and $D from the loop above.

Reply:
Yeah so in the case of auth, $D is auth(the last piece of the path) 
MODDOC=docs/integrate/modules/$(basename "$D") so its just basename (auth) + auth(last piece of path).md

This is to ensure in the `docs-repo` the tab for the page doesnt read `README` and reads `auth` as its not aesthetically pleasing